deploy/backplane/lpsre/rhoc/01-rhoc-lpsre-admins-cluster.ClusterRole.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-lpsre-rhoc-admins-project
```

Comment:
Just noticed the name of this file is inconsistent with the name of the ClusterRole.  The suffix of "cluster" implies permissions that will be bound at a cluster scope, but that is not the way this is used.  Please rename the file to "01-rhoc-lpsre-admins-project.ClusterRole.yaml" (/cluster/project/)